roofit/roofitcore/CMakeLists.txt

```diff
@@ -227,6 +227,7 @@ ROOT_STANDARD_LIBRARY_PACKAGE(RooFitCore
     RooFitLegacy/RooCatTypeLegacy.h
     RooFitLegacy/RooCategorySharedProperties.h
     RooFitLegacy/RooTreeData.h
+    RunContextTracker.h
```

Comment:
Maybe move above the legacy block? Not really necessary, though.

Reply:
Done.